packages/core/src/git.ts

```diff
@@ -50,11 +50,11 @@ const makePrBodyIdentifier = (context: string) =>
 // A class to interact with the local git instance and the git remote.
 // currently it only interfaces with GitHub.
 export default class Git {
+  readonly ghub: Octokit;
```

Comment:
A tiny nit here, but could we expand this to just be `github`? It's only a few more characters and with typescript/code we get autocompletion so there's really _not_ more to type.